stubs/pytest-snapshot/pytest_snapshot/plugin.pyi

```diff
@@ -0,0 +1,27 @@
+import re
+from collections.abc import Generator
+from pathlib import Path
+from types import TracebackType
+from typing import Any
+
+import _pytest.config.argparsing
+import pytest
+
+PARAMETRIZED_TEST_REGEX: re.Pattern[str]
+
+def pytest_addoption(parser: _pytest.config.argparsing.Parser) -> None: ...
+@pytest.fixture
+def snapshot(request: pytest.FixtureRequest) -> Generator[Snapshot, Any, None]: ...
+
+class Snapshot:
+    def __init__(self, snapshot_update: bool, allow_snapshot_deletion: bool, snapshot_dir: Path): ...
+    def __enter__(self): ...
+    def __exit__(
+        self, exc_type: type[BaseException] | None, exc_val: BaseException | None, exc_tb: TracebackType | None
+    ) -> None: ...
+    @property
+    def snapshot_dir(self) -> Path: ...
+    @snapshot_dir.setter
+    def snapshot_dir(self, value: str | Path) -> None: ...
+    def assert_match(self, value: str | bytes, snapshot_name: str | Path) -> None: ...
+    def assert_match_dir(self, dir_dict: dict[Any, Any], snapshot_dir_name: str | Path) -> None: ...
```

Comment:
We now require each use of `Any` to be documented with the accepted types/usage. If you are unsure what the correct types are, you can use `Incomplete` (from `_typeshed`) – which is an alias of `Any` – instead.